src/main/java/com/exactpro/epfast/template/Sequence.java

```diff
@@ -0,0 +1,19 @@
+package com.exactpro.epfast.template;
+
+import java.util.List;
+
+public interface Sequence extends Instruction {
+
+    IdentityRef getFieldId();
+
+    Presence getPresence();
+
+    Dictionary getDictionary();
```

Comment:
it can be removed. The idea is to keep dictionary only where we use it. In schema it's where both dictionary and key attributes are present on element